pkg/model/model.go

```diff
@@ -40,6 +41,8 @@ type Model struct {
 	typeDefs     []*TypeDef
 	typeImports  map[string]string
 	typeRenames  map[string]string
+	// Metadata for the service
+	meta *ackmetadata.ServiceMetadata
```

Comment:
Do we need `*ackmetadata.ServiceMetadata` in `ackmode.Model` and `multiversion.APIVersionManager`? what are the use cases when generating a one api version (`ack-generate apis`)?

Reply:
True. I had this in before your last PR, I'll remove it again.